go.mod

```diff
@@ -79,6 +79,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.1
 )
 
+require github.com/longhorn/longhorn-spdk-engine v0.0.0-20230627161406-64cb30ad3cf1 // indirect
+
```

Comment:
Not sure why this `require` statement is separated out?

Reply:
Not sure. Fixing in rebase/amend.